daft/ai/_expressions.py

```diff
@@ -58,6 +60,23 @@ def __call__(self, text_series: Series) -> list[Label]:
         return self.text_classifier.classify_text(text, labels=self.labels) if text else []
 
 
+class _ImageClassificationExpression:
+    """Function expression implementation for a ImageClassifier protocol."""
+
+    image_classifier: ImageClassifier
+    labels: list[Label]
+
+    def __init__(self, image_classifier: ImageClassifierDescriptor, labels: list[Label]):
+        self.image_classifier = image_classifier.instantiate()
+        self.labels = labels
+
+    def __call__(self, image_series: Series) -> list[Label]:
+        from PIL import Image
+
+        images = [Image.fromarray(image) for image in image_series]
+        return self.image_classifier.classify_image(images, labels=self.labels) if images else []
```

Comment:
Very minor -- not strictly necessary for review -- you can avoid the allocation of an empty list (`images`) and instead just directly return a single empty list:
```
if len(image_series) == 0:
  return []
images = ...
return self.image_classifier....
```